### ISSUE TITLE
Add noise when training

### ISSUE THREAD

**Issue by ShyPooper** [zhengchen1999/DOVE#22]
I've tried adding a rand noise to lq_latent before predicting noise (set the training config sr_noise_step as 399)，but got terrible results.

         #Noise Step (Select)
        if self.args.noise_step != 0:
            add_timesteps = torch.full(
                (batch_size,),
                fill_value=self.args.noise_step,
                dtype=torch.long,
                device=self.accelerator.device,
            )

            # Add noise to latent
            noise = torch.randn_like(lq_latent)
            lq_latent = self.components.scheduler.add_noise(lq_latent, noise, add_timesteps)

I wonder whether this issue resulted from my insufficient training, or whether the noise-adding method itself inherently leads to this situation. Currently, after adding noise, I haven’t changed any other training parameters (e.g., batch size, number of training steps, learning rate, etc.). 

Has anyone tried training with adding noise ? If so, what additional action were made after adding noise? And how did the final training results turn out?

**Comment by zhengchen1999** [comment]
Hello, in one-step VSR, we do not recommend adding extra noise, because one-step denoising itself is not well suited to handling overly severe noise.
If you do want to incorporate noise, you may further adopt certain losses, such as score-based losses (e.g., DMD), or use 2–3 denoising steps instead.
